integration_tests/jenkins_jobs/03_setting_distribution.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+cd $HOME/friday/integration_tests && \
+pip3 install -r ./requirements.txt && \
+PATH=$PATH:$HOME/go/bin && \
+python3 config_setting.py
+
+scp -i ~/ci_nodes.pem ~/.nodef/config/genesis.json opc@132.145.83.49:~/.nodef/config
```

Comment:
@JoowonYun 
> The build information must be the same wasm file. In the current master version ...

Could you please tell me the reason?
In my knowledge, in `make install`, WASMs are built in each node based on EE repository, and copied to contracts directory.
It means that WASMs seemed to be built in same version, so it doesn't need to be distributed.

Reply:
system_contract is ok, because of genesis have it. But another WASM(transfer, bond, unbond) must be same wasm file. This is a problem if you do not use the same file. For example, it includes build information, time, etc., which creates a different state.

But when the stored contract goes ahead, it's unnecessary.

Reply:
https://github.com/hdac-io/friday/pull/74/commits/6a2ee68761d6d3c6f9f6df608e4e229c50c54982

Reply:
And https://github.com/hdac-io/friday/pull/74/commits/e89f5a959fbb79da4887f394d1d908d28fcc7e75

If you want to reproduce it, you may edit `03_(Do not trigger yourself) Setting distribution
` job and block `scp contracts` part.
The shell script in this commit is just for backup, not actual Jenkins job execution.